tests/NuGet.Jobs.Catalog2Registration.Tests/Hives/HiveMergerFacts.MergeAsync.cs

```diff
@@ -0,0 +1,411 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Versioning;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace NuGet.Jobs.Catalog2Registration
+{
+    public partial class HiveMergerFacts
+    {
+        public class MergeAsync : Facts
```

Comment:
Please, don't name test classes exactly like methods, it messes up Ctrl+T search in VS: classes pop up in search higher than methods and in most cases you'd want method.

Reply:
Sounds good. Would something like `TheMergeAsyncMethod` be better?

Reply:
Did that!